src/main/scala/com/github/traviscrawford/spark/dynamodb/DynamoDBRelation.scala

```diff
@@ -11,12 +11,13 @@ import com.amazonaws.services.dynamodbv2.document.Table
 import com.amazonaws.services.dynamodbv2.document.spec.ScanSpec
 import com.amazonaws.services.dynamodbv2.model.ReturnConsumedCapacity
 import com.amazonaws.services.dynamodbv2.xspec.ExpressionSpecBuilder
-import org.apache.spark.Logging
+import org.apache.log4j.LogManager
 import org.apache.spark.rdd.RDD
 import org.apache.spark.sql.Row
 import org.apache.spark.sql.SQLContext
 import org.apache.spark.sql.sources._
 import org.apache.spark.sql.types._
+import org.spark_project.guava.util.concurrent.RateLimiter
```

Comment:
Please take a look at https://github.com/traviscrawford/spark-dynamodb/pull/11 and let me know if you have any comments/suggestions.